### ISSUE TITLE
NIST ACVP server was down 5/19/2022

### ISSUE THREAD

**Issue by ehanson12** [usnistgov/ACVP-Server#207]
Hello,

UPDATED to add DEMO Server.

The NIST ACVP DEMO server was down for the entire day of 5/19/2022. I believe the outage started somewhere around 22:00 Eastern time on 5/18/22. Today it is up but running slowly.

Best Regards - Eric

**Comment by livebe01** [comment]
Hi @ehanson12, this is in reference to the Demo server?

**Comment by ehanson12** [comment]
Yes,

I should have mentioned that...

**Comment by jarnold01** [comment]
hi @ehanson12 - yes, I can confirm that there was an issue with the Demo environment during the dates/times you specified. This was due to an upstream network issue which I have been working with our network teams with to get resolved, but it is very intermittent. The perceived slowness you experienced was likely just the system working through the backlog of submissions once I restored the network communication as the tasks were being processed. 

Thanks for reporting this.
